frontend/src/bundles/view-template-page/view-template-page.tsx

```diff
@@ -0,0 +1,72 @@
+import { useNavigate, useParams } from 'react-router-dom';
+
+import { Footer, Header, RegularButton } from '../common/components/components';
+import { AppRoute, ButtonType, ButtonVariant } from '../common/enums/enums';
+import {
+    useAppDispatch,
+    useCallback,
+    useEffect,
+    useState,
+} from '../common/hooks/hooks';
+import { TemplateEditor } from '../edit-template/components/template-editor/template-editor';
+import { actions as templateActions } from '../edit-template/store/edit-template.store';
+import { type TemplateDto } from '../templates-page/types/types';
+import { ToastType } from '../toast/enums/show-toast-types.enum';
+import { showToast } from '../toast/helpers/show-toast';
+import styles from './styles.module.scss';
+
+const ViewTemplatePage: React.FC = () => {
+    const { id } = useParams();
+    const [template, setTemplate] = useState<TemplateDto | null>(null);
+    const navigate = useNavigate();
+
+    const dispatch = useAppDispatch();
+
+    useEffect(() => {
+        if (id) {
+            dispatch(templateActions.getTemplateById(id))
+                .unwrap()
+                .then((template) => setTemplate(template))
+                .catch((error) => showToast(error, ToastType.ERROR));
+        }
+    }, [dispatch, id]);
+
+    const editTemplateHandler = useCallback(() => {
+        navigate(`${AppRoute.TEMPLATE_EDITOR}/${template?.id}`);
+    }, [navigate, template]);
+
+    const returnBackHandler = useCallback(() => {
+        navigate(AppRoute.TEMPLATES);
+    }, [navigate]);
+
+    return (
+        <>
+            <Header />
+            <section className={styles.template_section}>
+                {template && (
+                    <TemplateEditor settings={template.templateSettings} />
```

Comment:
Consider using an image for this, templateEditor is too heavy

Reply:
Good idea. Gonna try